nailgun/entities.py

```diff
@@ -2031,6 +2031,17 @@ def __init__(self, server_config=None, **kwargs):
         }
         super(ContentCredential, self).__init__(server_config, **kwargs)
 
+    def update_payload(self, fields=None):
+        """Override the method to only update fields that actually can be updated
+        """
+        updatable_fields = set.intersection({'name', 'content_type', 'content'},
+                                            self.get_values().keys())
+        if fields is None:
+            fields = updatable_fields
+        else:
+            fields = set.intersection(set(fields), updatable_fields)
+        return super().update_payload(fields=fields)
```

Comment:
Because ContentCredential has multiple inheritence, and because the method that you want to call here is in the last class that is inherited, I think it would be better to provide the type here to `super`, in order to be as explicit as possible and avoid the potential for this to resolve and call the wrong method from one of the other inherited classes.

`super(EntityUpdateMixin, self).update_payload()`

Reply:
You're right. I'll fix it in bulk after I get all the feedback.